db/database.go

```diff
@@ -484,21 +483,11 @@ func NewDatabaseContext(dbName string, bucket base.Bucket, autoImport bool, opti
 
 		for name, provider := range options.OIDCOptions.Providers {
 			if provider.Issuer == "" || base.StringDefault(provider.ClientID, "") == "" {
-				base.WarnfCtx(logCtx, "Issuer and ClientID required for OIDC Provider - skipping provider %q", base.UD(name))
+				// TODO: this duplicates a check in DbConfig.validate to avoid a BWC issue
```

Comment:
I'm not sure BWC is a commonly used acronym. At least not one that I've ever ran into before. Can we just keep it English? 😄

Reply:
```suggestion
				// TODO: this duplicates a check in DbConfig.validate to avoid a backwards compatibility issue
```